src/actions/guides/getting-started/installing.cr

```diff
@@ -225,6 +226,25 @@ class Guides::GettingStarted::Installing < GuideAction
 
     There are other installation methods available in [Postgres CLI tools docs](https://postgresapp.com/documentation/cli-tools.html)
 
+    ### 1b. Password-less logins for local development
```

Comment:
Yes, we probably should call out Linux and I will make that change -- I wasn't 100% sure if it applied to new macOS setups or not.

Reply:
Singling out Linux now.  I double-checked Homebrew formula for PostgreSQL and it is setting up the `pg_hba.conf` file to be a lot more lenient, `trust`ing all logged in users to connect to the server.  Homebrew's choice for defaults doesn't make a lot of sense for Linux since macOS is desktop 99% of the time while Linux is more likely to be a server for PostgreSQL deploys.  Hence the difference in user experience.

Reply:
One thing I would add.  I am a long-time Linux user as well as Linux on servers.  Linux as a desktop/development experience is fairly new for me and I would never deploy PostgreSQL without strong passwords on all the users.  That's what completely threw me here because I was used to macOS' defaults as well as simply operating in server-space the way a security conscious devop should.   Because of all the changes with the Crystal language, I suspected the issue resided in implementation rather than system configuration for many hours, before looking into postgresql itself.  That's the pain I'm hoping to solve for the next Linux-based developer with this PR.